### ISSUE TITLE
Add a contributor Chrome Custom tab

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/OpenFoodFacts-androidApp#190]
http://en.wiki.openfoodfacts.org/API#Contributors

**Comment by teolemon** [comment]
![screenshot_20161113-115826](https://cloud.githubusercontent.com/assets/1689815/20257195/3d6a6e00-aa48-11e6-8eb0-f5666823dfd8.png)
The icon Google Maps uses is nice

**Comment by herau** [comment]
@teolemon you add API links but you want to open the open food facts in a CCT right ?

**Comment by teolemon** [comment]
Yes, I changed it this morning. Sorry for the confusion. Native is not a priority for those kind of pages.

**Comment by herau** [comment]
waiting the merge of #218 before starting the code (needed code for the chrome custom tab)

**Comment by teolemon** [comment]
Limiting to just contributor. Product history belongs to the product view.
